### PR TITLE
Refactor Display and Win Mechanisms To Simplify Code

### PR DESCRIPTION
**Pull request by ajongaro**
New horizontal_win methods
New vertical_win methods
New pretty_print method
and other refactors.

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -66,10 +66,53 @@ def tie?
   end
 
   # return formatted board for printing to terminal
-  def display_board 
-    "#{@header}\n#{row_1}\n#{row_2}\n#{row_3}\n#{row_4}\n#{row_5}\n#{row_6}"
+  def pretty_print
+    array = ["ABCDEFG\n"]
+    (1..6).to_a.each do |number|
```
Comment (BRohrig) [inline lib/board.rb:71]: this is much cleaner looking, glad we went over this, and it was easier than expected to refactor using what we already had

File: lib/board.rb [hunk 2]
```diff
@@ -66,10 +66,53 @@ def tie?
...
+      end
+      array << "\n"
+    end
+    array.join("")
+  end
+
+  def search_for_winner
+    return true if horizontal_win
+    return true if vertical_win
+    # return true if diag_win
```
Comment (BRohrig) [inline lib/board.rb:83]: as we talked about i will fold the diagonal method into what we already have and test shortly

**Review by BRohrig — APPROVED** [review]

File: lib/board.rb (hunk 1 above)
Comment (ajongaro) [reply lib/board.rb:71]: Thanks again for your help as we worked through this, great team vibes.

File: lib/board.rb (hunk 2 above)
Comment (ajongaro) [reply lib/board.rb:83]: Excellent, thank you! Great work today.

**Pull request merged**